// anime-actions/lib/functions.js
const resolve = require('./resolve.js')

module.exports = {

    baka: function () {
        return resolve.get('baka');
    },
    blush: function () {
        return resolve.get('blush')
    },
    shoot: function() {
        return resolve.get('shoot')
    },
    cuddle: function () {
        return resolve.get('cuddle')
    },
    bite: function () {
        return resolve.get('bite')
    },
    dance: function () {
        return resolve.get('dance')
    },
    slap: function() {
        return resolve.get('slap')
    },
    bonk: function() {
        return resolve.get('bonk')
    },
    bully: function() {
        return resolve.get('bully')
    },
    hug: function() {
        return resolve.get('hug')
    },
    kiss: function() {
        return resolve.get('kiss')
    },
    waifu: function() {
        return resolve.get('waifu')
    },
    pat: function() {
        return resolve.get('pat')
    },
    eating: function() {
        return resolve.get('eating')
    },
    handshake: function() {
        return resolve.get('handshake')
    },
    lurk: function() {
        return resolve.get('lurk')
    },
    wallpaper: function() {
        return resolve.get('wallpaper')
    },
    happy: function() {
        return resolve.get('happy')
    },
    wink: function() {
        return resolve.get('wink')
    },
    kill: function() {
        return resolve.get('kill')
    },
    highfive: function() {
        return resolve.get('highfive')
    },
    wave: function() {
        return resolve.get('wave')
    },
    cry: function() {
        return resolve.get('cry')
    },
    smile: function() {
        return resolve.get('smile')
    },
    punch: function() {
        return resolve.get('punch')
    },
    kick: function() {
        return resolve.get('kick')
    },
    poke: function() {
        return resolve.get('poke')
    },
    goodnight: function() {
        return resolve.get('goodnight')
    },
    confused: function() {
        return resolve.get('confused')
    },
    sad: function() {
        return resolve.get('sad')
    },
    thinking: function() {
        return resolve.get('thinking')
    },
    yes: function() {
        return resolve.get('yeah')
    },
    yeet: function() {
        return resolve.get('yeet')
    },
    stare: function() {
        return resolve.get('stare')
    },
    bored: function() {
        return resolve.get('bored')
    },
    scream: function() {
        return resolve.get('scream')
    },
    nervous: function() {
        return resolve.get('nervous')
    },
    nsfw: {
        anime: {
            ass: function () {
                return resolve.get('ass')
            },
            hentai: function () {
                return resolve.get('hentai')
            },
            maid: function () {
                return resolve.get('maid')
            },
            gif: function () {
                return resolve.get('gif')
            },
            glass: function() {
                return resolve.get('glass')
            },
            milf: function() {
                return resolve.get('milf')
            },
            blowjob: function () {
                return resolve.get('blowjob')
            },
            feet: function () {
                return resolve.get('feet')
            },
            pussy: function () {
                return resolve.get('pussy')
            },
            uniform: function () {
                return resolve.get('uniform')
            },
            school: function () {
                return resolve.get('school')
            },
            boobs: function () {
                return resolve.get('boobs')
            },
            spank: function () {
                return resolve.get('spank')
            },
            thigh: function () {
                return resolve.get('thigh')
            },
            succubus: function() {
                return resolve.get('succubus')
            },
            zettaiRyouiki: function() {
                return resolve.get('zettai')
            },
            wallpaper: function() {
                return resolve.get('hwallpaper')
            }
        },
        real: {
            gif: function () {
                return resolve.get('rgif')
            },
            hd: function() {
                return resolve.get('fourk')
            },
            anal: function() {
                return resolve.get('ranal')
            },
            
            ass: function() {
                return resolve.get('rass')
            },
            pussy: function() {
                return resolve.get('rpussy')
            },
            thigh: function() {
                return resolve.get('rthigh')
            },
            boobs: function() {
                return resolve.get('rboobs')
            }
        }    
    }
}
